docs/advanced/witness.md

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 8
+sidebar_label: Witness Node
+title: "Witness Node"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.1/advanced/witness/"/>
+</head>
+
+_Available as of v1.3.0_
+
+On the edge cases, the witness node plays a specific role in maintaining the cluster's high availability. The witness node has the responsibility to maintain the quorum of the etcd cluster. Although the witness node did not store any user data, the hardware requirement must also be considered. The lower performance hardware will also cause the potential risk of the etcd cluster. You can refer to this KB [Slow etcd performance (performance testing and optimization)](https://www.suse.com/support/kb/doc/?id=000020100) to know more about the performance issue of the etcd cluster. Also, here is a hardware recommendation for the etcd nodes from the [article](https://etcd.io/docs/v3.3/op-guide/hardware/).
+
+Harvester now supports the witness node involved cluster. You can enable the witness node when joining the cluster.
+For information about the role management, please refer to [role management](../TBD.md).
+
+
+:::note
+
+The current limitation is that the witness node can only be enabled when joining the cluster. Also, the cluster can only have one witness node.
+
+:::
+
+## Create Harvester cluster with witness node
+
+When you create a new cluster, you can choose the witness role when joining the cluster.
+For example, you create a cluster with `harvester-node-0`, `harvester-node-1` and `harvester-node-2`.
+
+Choose the `harvester-node-1` as the witness node when joining the cluster. After the cluster is created, you would see the `harvester-node-1` only has the etcd capability as follows:
+
+```
+NAME↑               STATUS   ROLE                         VERSION               PODS     CPU      MEM    %CPU    %MEM    CPU/A    MEM/A AGE        
+harvester-node-0    Ready    control-plane,etcd,master    v1.27.10+rke2r1         70    1095    10143      10      63    10000    15976 4d13h      
+harvester-node-1    Ready    etcd                         v1.27.10+rke2r1          7     258     2258       2      14    10000    15976 4d13h      
+harvester-node-2    Ready    control-plane,etcd,master    v1.27.10+rke2r1         36     840     6905       8      43    10000    15976 4d13h      
+```
+
+You can see the `harvester-node-1` use less of resource to running as expection. You need to satisfy the three-node requirement, and the promote controller will promote the second and third nodes.
```

Comment:
Hi @jillian-maroket, sorry to confuse you.

Here, I would like to express that we need to create a 3-node cluster and then promote the second and third nodes to witness nodes or management nodes, as an example.

Reply:
Thanks. What I don't understand is why we are mentioning the three-node requirement and the need to promote nodes directly after the example. In any case, you can replace your sentence with something like this:

"Because the cluster must have three nodes, the promote controller will promote the other two nodes."

Reply:
Thanks!
I rewrote this part. Please help recheck it!